ubuntu/16.04/usr/local/share/bootstrap/common_functions.sh

```diff
@@ -69,11 +69,6 @@ is_chown_forbidden() {
   return $?
 }
 
-is_vboxsf_mountpoint() {
-  grep -q "/app vboxsf" /proc/mounts
-  return $?
-}
-
```

Comment:
By CP specific i meant these are the CP Dockerfiles so they are specific to this project :) ... ideally a dev should just do 'docker-compose up -d' and be done.

If theres an alternate approach with setfacl, ive never needed to use it so am not that familiar with. is there a timeframe? if its a maybe or at the back of the backlog would one of the above approaches be okay?